core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -98,8 +98,8 @@ <h4 class="bold-title">
                         </section>
                         <section class="col-lg-2 col-md-2 pad-left0 runtime-option">
                             <div class="radio-container">
-                                <input type="radio" name="runtime" id="go" [value]="go1.x" [checked]="runtime == 'go1.x'" (change)="onSelectionChange('go1.x')">
-                                <label for="go1.x"><span class='outer'></span><span class="background"></span>Go 1.x</label>
+                                <input type="radio" name="runtime" id="go" [value]="'go1.x'" [checked]="runtime == 'go1.x'" (change)="onSelectionChange('go1.x')">
+                                <label [for]="'go'"><span class='outer'></span><span class="background"></span>Go 1.x</label>
```

Comment:
Why are we reverting this change?

Reply:
I just found out this option is not selectable without this change when I do the testing. Apologies that I didn't try to click the radio button yesterday when I test it. This time it's thoroughly tested.

Reply:
A little confused though.  
How are other runtime options working? Is it something related to usage of period (version) in the string (`python` vs `go1.x`)?

Reply:
@zoexiong, Tried fixing with your solution. HTML was parsing without any error but radio button wasn't working as expected.
I think attribute values should be assigned as following(when you are using period/version):
id="go1.x"
[value] = "'go1.x'"
[for]="'go1.x'"